source/logHandler.py

```diff
@@ -141,15 +141,21 @@ def getCodePath(f):
 	return ".".join(x for x in (path,className,funcName) if x)
 
 
+_onErrorSoundRequested = None
+
+
+def getOnErrorSoundRequested():

```

Comment:
You'll need to import `extensionPoints` using `if TYPE_CHECKING`.

```suggestion
def getOnErrorSoundRequested() -> "extensionPoints.Action":
```

Reply:
Fixed in 4472a99ba88b42146f877b434b695bb998c6475c